build.gradle

```
@@ -111,7 +111,7 @@ allprojects {
 
     repositories {
         maven { url "https://dl.bintray.com/hmcts/hmcts-maven" }
-        maven { url 'https://repo.spring.io/libs-milestone' }
+//        maven { url 'https://repo.spring.io/libs-milestone' }
```

```suggestion
        maven { url 'https://repo.spring.io/milestone' }
```
In line with the authentication changes that took place on 21st of Jan according to [this notice](https://spring.io/blog/2020/10/29/notice-of-permissions-changes-to-repo-spring-io-fall-and-winter-2020) we should probably follow the recomendation to change from `libs-milestone` to `milestone`.